homework102.py

```diff
@@ -0,0 +1,101 @@
+"""Homework102:Functions. Writing and implementing programs."""
+
+
+# Строки с заданным символом
+# Напишите программу, которая бы работала следующим образом - находила символ
+# "#" и если этот символ найден - удаляла предыдущий символ из строки. Ваша
+# задача обработать строки с "#" символом
+
+def remove_excessive_symbols(str_with_hash: str) -> str:
+    """
+    Removes the character preceding each '#' in the string.
+    """
+    while True:
+        try:
+            index = str_with_hash.index('#')
+            if index > 0:
+
+                str_with_hash = str_with_hash[:index - 1] + str_with_hash[
+                                                            index + 1:]
```

Comment:
remove extra spaces, I mean before and after else, after index > 0, it's not needed here

Reply:
done